test/fw/ptl/utils/plugins/ptl_test_db.py

```diff
@@ -1634,7 +1650,7 @@ def write(self, data, logfile=None):
     def close(self, result=None):
         if result is not None:
             dur = str(result.stop - result.start)
-            self.jdata['test_summary']['test_duration'] = dur
+            self.jdata['tests_duration'] = dur
```

Comment:
why did you changed this?

Reply:
this duration is the total time taken by all repetition of all run tests. So I keep it after all repetition.

Reply:
Can you please post before and after change json report?